SciAnalysis/tools.py

```diff
@@ -192,9 +192,9 @@ def run(self, infiles=None, protocols=None, output_dir=None, force=False, ignore
             except OSError:
                 print('  ERROR (OSError) with file {}.'.format(infile))
 
-            #except Exception as exception:
-                # Ignore errors, so that execution doesn't get stuck on a single bad file
-                #print('  ERROR ({}) with file {}.'.format(exception.__class__.__name__, infile))
+            except Exception as exception:
```

Comment:
i would comment this back in. looks like the intention is to allow this to run in batch

Reply:
Comment which lines back in? I think this is mainly to deal with some abnormal numpy behavior. Sometimes, depending on numpy version, I get division by zero alert, which stops the batch processing. This would allow the processing to continue on the rest of the input files.

Reply:
ok that sounds good